github/Deployment.py

```diff
@@ -150,6 +167,14 @@ def get_statuses(self):
             self._requester,
             self.url + "/statuses",
             None,
+            headers={
```

Comment:
This is repeated a fair amount -- would you consider a static method in this class that returns the string?

Reply:
This is now implemented.